config/config.go

```diff
@@ -38,9 +38,9 @@ type TypeConfig struct {
 
 // PubkeyConfig will map the public key configuration
 type PubkeyConfig struct {
-	Length          int
-	Type            string
-	SignatureLength int
```

Comment:
Why did you delete this?

Reply:
leave the length there.

Reply:
I deleted only `SignatureLength int` because it's not used.